content/docs/operating/security.md

```diff
@@ -84,7 +86,8 @@ configuration file.
 Any user with access to the Pushgateway HTTP endpoint can create, modify and
 delete the metrics contained within. As the Pushgateway is usually scraped with
 `honor_labels` enabled, this means anyone with access to the Pushgateway can
-create any time series in Prometheus.
+create any time series in Prometheus, including impersonating other targets or
```

Comment:
This isn't possible.

Reply:
How so? This was written in this file before my PR; see above.

Reply:
You cannot generate alerts this way. The current wording is fine.